### DISCUSSION TITLE
Source based AUR package says it's up to date which is not correct?

### DISCUSSION THREAD

**Discussion by GrabbenD** [Morganamilo/paru#1060] [Q&A] [answered]
AUR package `hyprland-git` (https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=hyprland-git) is downloaded and built directly from Github from my understanding.

What's confusing to me is that the package description in AUR shows the current version as `a0b675ec` **(2 weeks old)** but after downloading it, I get a newer version `d0367d85` **(1 week old)** and I'm guessing it's because the page is only updated when the maintainer updates contents of the AUR repo like `./PKGBUILD`:

```bash
# paru -Q hyprland-git
hyprland-git 0.31.0.r3529.d0367d85-1
```

However, Github commit [history](https://github.com/hyprwm/Hyprland/commits/main) shows  `1c9d6b9` as the latest commit **(2 hours ago)** but I can't seem to update?

```bash
# paru -Sy hyprland-git
warning: hyprland-git-0.31.0.r3529.d0367d85-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)              Old Version              New Version              Net Change

cachyos-v3/hyprland-git  0.31.0.r3529.d0367d85-1  0.31.0.r3529.d0367d85-1    0.00 MiB
```

**Comment by Morganamilo** [comment, marked as answer]
Looks like you have local repos on so you need to specify from the aur with `paru -Sa hyprland-git`

**Reply by GrabbenD** [reply]
Sometimes you go blind when you're tired 🙂 
CachyOS had their own version of `hyprland-git` .. (`cachyos-v3/hyprland-git`)
Everything is correct now!

```console
$ paru -R hyprland-git
$ paru -Sy aur/hyprland-git

$ paru -Q hyprland-git
hyprland-git 0.31.0.r61.ab5497a0-1
```
